BraveWallet/Chart/LineChartView.swift

```diff
@@ -265,6 +266,69 @@ extension CGPoint {
   }
 }
 
+// MARK: - Accessibility
+
+extension View {
```

Comment:
This in particular is not private no, because it can be applied to any `LineChartView` used. I could technically extend only LineChartView's but it is weird to do that in SwiftUI. The actual chart descriptor below is already private